lang/en_US/validation.php

```diff
@@ -86,7 +86,7 @@
         'array' => 'The :attribute must not have more than :max items.',
         'file' => 'The :attribute must not be greater than :max kilobytes.',
         'numeric' => 'The :attribute must not be greater than :max.',
-        'string' => 'The :attribute must not be greater than :max characters.',
+        'string' => 'Your :attribute is too long. It should be no more than :max characters.',
```

Comment:
Why change this? It's inconsistent with the other messages.

Reply:
Agree- mostly at the request of the corresponding ticket:

> To finish this report, IF I am not asking too much, may we have the exception message a bit less technical, please? That "The Body must not be greater than 2000 characters" is ok for me, but it would be better (and more user-friendly) for just showing a message like "Your message is too LONG! It must not be greater than 2000 characters!" or similar. 

I have no strong opinion.

Reply:
"Your Body is too long. It should be no more than 2000 characters." is not any clearer. I think it's the `:attribute` that is the most confusing.

Reply:
Let's punt then; I'm not sure what side effects tweaking this further could have, and it may wind up growing to justify its own PR. I'm going to revert this change.

Reply:
In matter of changing validation message inconsistency, that's why I said that above:

> The user will understand what "Motto" means, but maybe not what "Body" means - unless if Textarea name attribute gets changed.

If I recall correctly, `:attribute` takes the `name` attribute of HTML element that was used on. So, just changing the Textarea name would suffice as the user will understand better what is going on.